src/components/Ranking/SidoRankList.tsx

```diff
@@ -25,27 +24,25 @@ const SidoRankList = ({ sortType }: SidoRankListProps) => {
           );
         }
       },
-      refetchOnWindowFocus: false,
       keepPreviousData: true,
       staleTime: 1000 * 60 * 5,
+      suspense: true,
     }
   );
 
   return (
     <>
-      {!sidoDustInfos && <Spinner />}
-      {sidoDustInfos &&
-        sidoDustInfos.map((sido, sidoIndex) => (
-          <SidoRankItem
-            key={sido.sidoName}
-            rank={sidoIndex + 1}
-            sidoName={sido.sidoName}
-            fineDustScale={sido.fineDustScale}
-            ultraFineDustScale={sido.ultraFineDustScale}
-            fineDustGrade={sido.fineDustGrade}
-            ultraFineDustGrade={sido.ultraFineDustGrade}
-          />
-        ))}
+      {sidoDustInfos?.map((sido, sidoIndex) => (
+        <SidoRankItem
+          key={sido.sidoName}
+          rank={sidoIndex + 1}
+          sidoName={sido.sidoName}
+          fineDustScale={sido.fineDustScale}
+          ultraFineDustScale={sido.ultraFineDustScale}
+          fineDustGrade={sido.fineDustGrade}
+          ultraFineDustGrade={sido.ultraFineDustGrade}
+        />
```

Comment:
```suggestion
        <SidoRankItem
          key={sido.sidoName}
          rank={sidoIndex + 1}
          sido={sido}
        />
```
이렇게 하는게 더 깔끔할 것 같네요!